README.md

```diff
@@ -103,15 +103,25 @@ Before you start make sure you have installed:
 
 ### Compatibility
 
-#### MongoDB Support
-Parse Server is continuously tested with the most recent releases of MongoDB to ensure compatibility. The rests run against the latest patch version of each MongoDB release. We follow the [MongoDB support schedule](https://www.mongodb.com/support-policy) and only test against versions that are officially supported by MongoDB and have not reached their end-of-life date yet.
+#### Node.js
+Parse Server is continuously tested with the most recent releases of Node.js to ensure compatibility. We follow the [Node.js Long Term Support plan](https://github.com/nodejs/Release) and only test against versions that are officially supported and have not reached their end-of-life date yet.
+
+| Version    | Latest Patch Version | End-of-Life Date | Compatibility      |
+|------------|----------------------|------------------|--------------------|
+| Node.js 10 | 10.23.2              | April 2021       | ✅ Fully compatible |
+| Node.js 12 | 12.20.1              | April 2022       | ✅ Fully compatible |
+| Node.js 14 | 14.15.4              | April 2023       | ✅ Fully compatible |
```

Comment:
Should we update the table?

Reply:
Ah, yes, added Node 15. Funny enough, end of life is in June already.